core/src/main/java/com/scalar/db/transaction/consensuscommit/CrudHandler.java

```diff
@@ -23,30 +23,33 @@
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
-import javax.annotation.concurrent.ThreadSafe;
+import javax.annotation.concurrent.NotThreadSafe;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@ThreadSafe
+@NotThreadSafe
```

Comment:
Just a question. What makes this class not thread-safe?
(maybe it's good to be noted.)

Reply:
This was originally not thread-safe, but I have just corrected it.